src/main/java/no/unit/nva/model/Role.java

```diff
@@ -0,0 +1,34 @@
+package no.unit.nva.model;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.util.Arrays;
+
+public enum Role {
+    CREATOR("Creator");
+
+    private String value;
+
+    Role(String value) {
+        this.value = value;
+    }
+
+    @JsonValue
+    public String getValue() {
+        return value;
+    }
+
+    /**
+     * Lookup enum by value.
+     *
+     * @param value value
+     * @return  enum
+     */
+    public static Role lookup(String value) {
+        return Arrays
+                .stream(values())
+                .filter(nameType -> nameType.getValue().equalsIgnoreCase(value))
+                .findAny()
+                .orElseThrow(IllegalArgumentException::new);
```

Comment:
What about
`.orElseThrow(() -> new IllegalArgumentException(format("%s not a valid Role, expected one of: %s", value, stream(Role.values()).map(Role::toString).collect(joining(", ")))));
`